lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1655,7 +1656,10 @@ instance Arbitrary RewardAccount where
 
 instance Arbitrary Coin where
     -- No Shrinking
-    arbitrary = Coin <$> choose (0, 1_000_000_000_000_000)
+    arbitrary = genStrictlyPositiveCoin
+
+genStrictlyPositiveCoin :: Gen Coin
+genStrictlyPositiveCoin = Coin <$> choose (1, 1_000_000_000_000_000)
```

Comment:
Perhaps we can replace this with calls to `Cardano.Wallet.Primitive.Types.Coin.Gen.genCoinLargePositive`.

Reply:
Fixed in bce20553d283b5a13c5eeccb76e19d6f88aa7e16.